index.js

```diff
@@ -82,6 +83,7 @@ function Bankai (entry, opts) {
     if (self.optimize) {
       b.transform(unassertify, { global: true })
       b.transform(yoyoify, { global: true })
+      b.transform(envify, { global: true })
```

Comment:
I don't think we'd need to apply this as a global transform; running `process.env.NODE_ENV` checks in libraries seems like a practice that shouldn't be encouraged / might cause issues